enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -126,8 +126,8 @@ void setReportOnlyReachable(boolean reportOnlyReachable) {
   }
 
   @VisibleForTesting
```

Comment:
Is this just visible for testing if the name matters for correct XML processing? That is, is some pieve of Maven invoking this method when parsing the pom.xml?

Reply:
It's just visible for testing. Maven enforcer rule  plugin does not use the setter. (So the setter name change was not directly fixing the bug; but it's for consistency).